src/System.Drawing.Design.Primitives/src/System/Drawing/Design/ToolboxItem.cs

```diff
@@ -983,118 +959,19 @@ protected virtual object ValidatePropertyValue(string propertyName, object value
                     value = filter;
                     break;
 
-                case "IsTransient":
+                case nameof(IsTransient):
                     ValidatePropertyType(propertyName, value, typeof(bool), false);
                     break;
             }
             return value;
         }
 
-        /// <include file='doc\ToolboxItem.uex' path='docs/doc[@for="ToolboxItem.ISerializable.GetObjectData"]/*' />
-        /// <internalonly/> 
-        // SECREVIEW NOTE: we do not put the linkdemand that should be here, because the one on the type is a superset of this one
-        [SuppressMessage("Microsoft.Usage", "CA2240:ImplementISerializableCorrectly")]
         void ISerializable.GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
that method should also be deleted.

Reply:
I may need to keep method with empty body or throw not implemented exception here. Toolbox need to be derived from Iserializable interface.